src/QMCTools/gpaw4qmcpack

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
Add ".py" surffix.

Please add a README.md file at  `src/QMCTools` with documentation about the relation of python files added in this folder by the current PR.

Reply:
Good. I'll write something. Maybe the Pyscf methods should also be covered in the same README, but I don't know too much about them. I find it a little bit confusing that there are .cpp and .py under the same directory with very different installation mechanisms and documentation, but that's a broader question and probably not so important until we have many more python tools.

Reply:
You are only responsible for what you added.

Reply:
I wrote a bit of README documentation for the latest additions.